src/com/company/tests/BoardTest.java

```diff
@@ -0,0 +1,62 @@
+package com.company.tests;
+
+import com.company.*;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+public class BoardTest {
+    private OnEndGameListener mockOnEndGameListener;
+    private Board board;
+
+
+    @Before
+    public void onBefore() {
+        mockOnEndGameListener = mock(OnEndGameListener.class);
+        board = new Board(2, mockOnEndGameListener);
+    }
+
+    @After
+    public void onAfter() {
+
+    }
+
+    @Test
+    public void getSignTextTest() {
+        //Given
+        board.insertSign(PlayerSignType.X, 1, 1);
+        //When
+        boolean result = board.getSignText(1,1) == "X";
```

Comment:
Czemu `board.getSignText(1,1) == "X";`? 
W "When" nie powinno się jeszcze nic porównywać!

Poza `assertTrue` i `assertFalse` są też inne asserty. Pogooglaj.

Reply:
Chciałam osiagnąć boolean w zmiennej wynikowej. 
Zmieniłam teraz na string, bez porównania i z assertEquals w części Then

Reply:
Super. O to właśnie chodziło :D